text/0081-truncate.md

```diff
@@ -0,0 +1,105 @@
+# RFC: Return TiKV to some certain version
+
+## Summary
+
+This proposal introduces a method to return TiKV to some certain version, which could be useful when doing some lossy recovery.
+
+## Motivation
+
+The initial motivation is to do lossy recovery on TiKV when deploying TiDB cluster across DCs with DR Auto-sync method.
+
+When use DR Auto-sync method to deploy TiDB cluster in two DC and 
+
+- there is a network isolation between primary and DR DC
+- PD make the replica state of primary DC to be Async after a while
+- the primary DC is down
+
+Then the sync state of the cluster (from the DR DC's PD) is not dependable. In this case, data can be partially sent to the TiKV nodes in DR, and break the ACID consistency.
+
+Though it is impossible to do lossless recovery in this situation, we can recover the data on TiKV nodes to a certain version (`resolved_ts` in this case), which is ACID consistency.
+
+## Detailed design
+
+Basically, we need a method to return TiKV to a certain version (timestamp), we should add a new RPC to kvproto:
```

Comment:
Is it an online or offline operation? Can TiKV keep processing incoming requests during `truncate`?